### PR TITLE
RavenDB-16007 Spatial Map Colors & Icon Fixes

### PR DESCRIPTION
**Pull request by mateuszbartosik**

### CODE REVIEW

File: src/Raven.Studio/wwwroot/Content/css/icons.less [hunk 1]
```diff
@@ -462,7 +462,6 @@
 .make-icon(running-queries, "\ea1f");
 .make-icon(stats-running-queries, "\ea1f");
 .make-icon(manage-server-running-queries, "\ea1f");
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/Content/css/icons.less:464]: why icon removed?

Comment (mateuszbartosik) [reply src/Raven.Studio/wwwroot/Content/css/icons.less:464]: It had the same icon code as line 127:
`.make-icon(additional-assemblies, "\ea3a");`
and has been added previously by Rafal with the correct code in line 123:
`.make-icon(spatial-map-view, "\ea40");`

File: src/Raven.Studio/typescript/viewmodels/database/query/spatialQueryMap.ts [hunk 2]
```diff
@@ -86,7 +86,7 @@ class spatialQueryMap extends viewModelBase {
             { color: spatialPolygonModel.colors[index % spatialPolygonModel.colors.length] }));
 
         const circleArray = this.circlesLayer().map((circle, index) => L.circle([circle.latitude, circle.longitude],
-            { color: spatialCircleModel.colors[index % spatialCircleModel.colors.length], fillOpacity: 0.2, radius: circle.radius }));
+            { color: spatialCircleModel.colors[index % spatialCircleModel.colors.length], fillOpacity: 0.4, radius: circle.radius }));
```
Comment (Danielle9897) [inline src/Raven.Studio/typescript/viewmodels/database/query/spatialQueryMap.ts]: imho - fillOpacity 0.4 is too much - reduce back to 0.2

File: src/Raven.Studio/typescript/viewmodels/database/query/spatialQueryMap.ts [hunk 3]
```diff
@@ -86,7 +86,7 @@ class spatialQueryMap extends viewModelBase {
             { color: spatialPolygonModel.colors[index % spatialPolygonModel.colors.length] }));
```
Comment (Danielle9897) [inline src/Raven.Studio/typescript/viewmodels/database/query/spatialQueryMap.ts]: for consistency, pls add fillOpacity for polygons here as well
=>
`{ color: spatialPolygonModel.colors[index % spatialPolygonModel.colors.length], fillOpacity: 0.2 }));`

File: src/Raven.Studio/typescript/viewmodels/database/query/spatialQueryMap.ts (hunk 2 above)
Comment (mateuszbartosik) [reply src/Raven.Studio/typescript/viewmodels/database/query/spatialQueryMap.ts]: Reduced

File: src/Raven.Studio/typescript/viewmodels/database/query/spatialQueryMap.ts (hunk 3 above)
Comment (mateuszbartosik) [reply src/Raven.Studio/typescript/viewmodels/database/query/spatialQueryMap.ts]: Added

**Review by Danielle9897 — APPROVED** [review]

**Review by ml054 — APPROVED** [review]

**Pull request merged**